src/features/vision.feature

```diff
@@ -3,4 +3,38 @@ Feature: Test for Vision page
   Scenario: Open Vision Insurance
     Given the user is on the Vision page
     When the Vision Insurance product page is opened
-    Then the main Vision title and description should be correct
\ No newline at end of file
+    Then the main Vision title and description should be correct
+    And correct icon is displayed next to the Vision title
+    And the title and description of What is Vision Insurance? should be correct
+
+  Scenario: Breadcrumb links testing
+    Given the user is on the Vision page
+    And the page path elements have correct text
```

Comment:
What is page path element? Breadcrumb?

Reply:
Yes. Wrote that step before deciding how to call breadcrumbs.